config/short-url.php

```diff
@@ -17,6 +17,19 @@
     */
     'disable_default_route' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Enforce HTTPS in the destination URL
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify if the visitor is redirected to the HTTPS version
+    | of the destination URL. This can be particularly useful if you're allowing
+    | your web app users to create their own shortened URLS and don't want to
+    | use the secure() method in every occasion.
+    |
+    */
+    'enforce_https' => false,
```

Comment:
Probably best updating this to ``` true ``` so that we can encourage developers to use secure methods by default.